baselines/baseline_word_embeddings.py

```diff
@@ -0,0 +1,120 @@
+import argparse
+import json
+import torch
+from torch.utils.data import DataLoader
+from pathlib import Path
+from utils import BertExtractor, StaticEmbeddingExtractor
+from training_scripts.nearest_neighbour import NearestNeigbourRanker
+from utils.data_loader import extract_all_labels, StaticRankingDataset, ContextualizedRankingDataset
+import numpy as np
+
+
+def get_average_phrase(data):
+    """
+    :param data: dataset as dataloader
+    :return averages: list of averages of phrases
+    """
+    data_all = next(iter(data))
+    averages = []
+    for m, h in zip(data_all["w1"], data_all["w2"]):
+        averages.append(m.add(h))
```

Comment:
This computes the "average phrase" by taking the sum of modifier and head. this a valid baseline but it is not really the "average" if we define "average" as the real average or "centroid" of two vectors (that would require dividing the sum by two)..we can add have two methods, one for getting the sum and one for getting the average

Reply:
done
the option is specified in the config. I think that makes more sense